src/resources/Schema.ts

```diff
@@ -50,7 +49,7 @@ export class Schema {
   valdiateSchema(schema: string) {
     const errors = validateSDL(parse(schema));
     if (errors.length > 0) {
-      throw new ServerlessError(
+      throw new this.api.plugin.serverless.classes.Error(
```

Comment:
Align the way errors are raised with the rest of the repository e.g. https://github.com/sid88in/serverless-appsync-plugin/blob/e99c51ab10a257a051672f53c211bc2a6674bd32/src/resources/Resolver.ts#L83.
The `ServerlessError` import is no longer available starting with serverless 4.0.23.